docs/CubesterYT/KeySimulation.md

```diff
@@ -0,0 +1,43 @@
+# Key Simulation
+
+This extension allows you to simulate Key Presses and Mouse Clicks on the Project.
+
+***
```

Comment:
I'm relatively certain that the extension docs syntax require "---" instead of "***"

Reply:
I'll check it, if so, then I'll fix it.

Reply:
Nope, *** works.

Reply:
> Nope, *** works.

It may work for general markdown, but it's unclear if "---" works, and it's preferred since all other documentation uses it anyways.

Reply:
I already checked, because I use the dev server for TW, so I can see the docs in its formatted form, they are fine.

Reply:
@NexusKitten I'll also take up this suggestion, I might as well do it for consistency.